packages/terra-hookshot/tests/jest/HookshotDefaultExample.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Hookshot from '../../src/Hookshot';
+
+class HookshotDefaultExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.state = { isOpen: false };
+  }
+
+  handleButtonClick() {
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  handleRequestClose() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    const hookshotContent = (
+      <Hookshot.Content onRequestClose={this.handleRequestClose}>
+        <div style={{ height: '40px', width: '200px', backgroundColor: 'red' }}>Hookshot</div>
+      </Hookshot.Content>
+    );
+
+    return (
+      <div>
+        <Hookshot
+          content={hookshotContent}
+          contentAttachment="top center"
+          isEnabled
+          isOpen={this.state.isOpen}
+          targetRef={() => document.getElementById('hookshot-standard-button')}
+        />
```

Comment:
Using `children` as opposed to `content` make the usage and examples much more natural in my opinion.

```
<Hookshot
  contentAttachment="top center"
  isEnabled
  isOpen={this.state.isOpen}
  targetRef={() => document.getElementById('hookshot-standard-button')}
>
  {hookshotContent}
</Hookshot>
```

Reply:
I went with content in that we only take one element, and it's only ever expect it to be of one type. The goal was to draw attention to the fact that this element isn't being wrapped, so it wouldn't handle an array, and didn't want to introduce logic to ensure it's only one child. It also aligns with the other prop names, so it's clear that content itself is matched with the other content named props.

Reply:
Can't you enforce that with PropTypes though and say that it only supports one. I really think the API is important. This is really a pet peeve of mine as a consumer.